### PR TITLE
Create Helm Chart

### LINKED ISSUE #35: Helm chart
Assigned @caiohasouza

### PR DESCRIPTION
**Pull request by caiohasouza**
Fixed #35

### CODE REVIEW

File: deploy/charts/oldmonk/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: oldmonk
+version: 0.0.2
```
Comment (evalsocket) [inline deploy/charts/oldmonk/Chart.yaml]: make it 0.1.0 until the release

File: deploy/charts/oldmonk/templates/autoscaler.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{{- if .Values.queues }}
```
Comment (evalsocket) [inline deploy/charts/oldmonk/templates/autoscaler.yaml]: It's a deployment-specific Yaml. This helm chart should only contain controller deployment. 
for example if we want to scale `billing-worker` then `billing-work` project should at least have `deployment.yaml` and `oldmonk.yaml` (specific to queue) so they can share a single secret. And this chart becomes specific to controller deployment

File: deploy/charts/oldmonk/templates/secrets.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{{- if .Values.queues }}
```
Comment (evalsocket) [inline deploy/charts/oldmonk/templates/secrets.yaml]: Same as above. It is also related to deployment. Can be manage by deployment

File: deploy/charts/oldmonk/values.yaml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+## Node tolerations for alertmanager scheduling to nodes with taints
+## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+##
+tolerations: []
+
+## Node tolerations for alertmanager scheduling to nodes with taints
+## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+##
+queues: []
```
Comment (evalsocket) [inline deploy/charts/oldmonk/values.yaml]: We don't want them here. You can remove

File: deploy/charts/oldmonk/templates/_helpers.tpl [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+{{- print "extensions/v1beta1" -}}
+{{- else if semverCompare "^1.7-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- print "networking.k8s.io/v1" -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the appropriate apiVersion for ingress.
+*/}}
+{{- define "ingress.apiVersion" -}}
```
Comment (evalsocket) [inline deploy/charts/oldmonk/templates/_helpers.tpl]: Where are are using `ingress.apiVersion`

File: deploy/charts/oldmonk/Chart.yaml (hunk 1 above)
Comment (caiohasouza) [reply deploy/charts/oldmonk/Chart.yaml]: Done

File: deploy/charts/oldmonk/templates/autoscaler.yaml (hunk 2 above)
Comment (caiohasouza) [reply deploy/charts/oldmonk/templates/autoscaler.yaml]: Ok, file removed.

File: deploy/charts/oldmonk/templates/_helpers.tpl (hunk 5 above)
Comment (caiohasouza) [reply deploy/charts/oldmonk/templates/_helpers.tpl]: Ok, removed.

**Pull request merged**